utils/constants.tsx

```diff
@@ -1950,6 +1950,23 @@ export const InsightsCardTitles = {
     },
 };
 
+export const AsciiToUtf8Characters = {
+    a: ['à', 'á', 'â', 'ã', 'ä', 'å'],
+    e: ['è', 'é', 'ê', 'ë'],
+    i: ['ì', 'í', 'î', 'ï'],
+    o: ['ð', 'ò', 'ó', 'ô', 'õ', 'ö', 'ø'],
+    u: ['ü', 'û', 'ú', 'ù'],
+    A: ['À', 'Á', 'Â', 'Ã', 'Ä', 'Å'],
```

Comment:
Do we need the uppercase variations of these? The string we're passing into `asciiToUtf8Variants` is lower case, so we might be able to get away without them